src/behavioural-events/events-tracker.ts

```diff
@@ -0,0 +1,110 @@
+import type { SDKInitializedEvent, TrackedEvent } from "./event-types";
+import { v4 as uuid } from "uuid";
+import { RC_ENDPOINT, VERSION } from "../helpers/constants";
+import { HttpMethods } from "msw";
+import { getHeaders } from "../networking/http-client";
+import { defaultHttpConfig, type HttpConfig } from "../entities/http-config";
+import { ExponentialInterval } from "../helpers/exponential-interval";
+
+const MIN_INTERVAL_RETRY = 2_000;
+const MAX_INTERVAL_RETRY = 60_000;
```

Comment:
Thoughts on these timings? The first one will always be skipped so it will be series 4s, 8s, 16s, 32s, 60s, 60s, 60s, ...